README.md

```diff
@@ -238,7 +238,7 @@ API | Description | Auth | HTTPS | CORS |
 API | Description | Auth | HTTPS | CORS |
 |---|---|---|---|---|
 | [1Forge](https://1forge.com/forex-data-api/api-documentation) | Forex currency market data | `apiKey` | Yes | Unknown |
-| [apilayer fixer.io](http://fixer.io) | Exchange rates and currency conversion | `apiKey` | Yes | Unknown |
+| [apilayer fixer.io](http://fixer.io) | Exchange rates and currency conversion | `apiKey` | Yes (paid) | Unknown |
```

Comment:
```suggestion
| [apilayer fixer.io](https://fixer.io) | Exchange rates and currency conversion | `apiKey` | Yes | Unknown |
```
`Yes (paid)` is not a valid string. It can be `Yes`, `No` or `Unknown` from [doc](https://github.com/public-apis/public-apis/blob/master/CONTRIBUTING.md#formatting)

Reply:
Ah sorry. I only see it mentioned for the CORS field there.

Would it make sense to have other options for the HTTPS field? 
While fixer does support HTTPS, they only do so on their paid plans. The free API key won't work over HTTPS. They made this change sometime late Nov 2020

Reply:
This repo is for "free" APIs only. If the free-trier plan can be used only with `HTTP` I suggest changing `Yes` to `No` in the HTTPS column.

Reply:
gotcha :)

Reply:
Thanks, @duckle29!